File: 2021 DEC 14/Reverse each word in a given string.py
#User function Template for python3

class Solution:
    def reverseWords(self, s):
        # code here
        # code here
        a = s.split(".")
        for i in range(len(a)):
            a[i] = a[i][::-1]
        b = ".".join(a)
        return b

#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        s = input().strip()
        ob = Solution()
        ans = ob.reverseWords(s)
        print(ans)
# } Driver Code Ends

File: 2021 DEC 14/Reverse each word in a given string.cpp
// { Driver Code Starts
#include<bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
class Solution
{
  public:
    string reverseWords (string s)
    {
        //code here.
        string t, ans;
    for (auto c : s) {
        if (c == '.') {
            reverse(t.begin(), t.end());
            ans += t + "."; t.clear();
        }
        else t.push_back(c);
    }
    reverse(t.begin(), t.end());
    return ans + t;
    }
};

// { Driver Code Starts.
int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        string s; cin >> s;
        Solution ob;
        cout << ob.reverseWords (s) << endl;
    }
    return 0;
}

// Contributed By: Pranay Bansal
  // } Driver Code Ends
